### PR TITLE
Add benchmarks

### PR DESCRIPTION
**Pull request by jonathanchu33**
- Benchmarks for `_benchmark_defun_matmul_with_signature`
- Add CPU benchmark for `_benchmark_defun_args_matmul`, analogous to GPU benchmark in line 686. Docstring copied directly; connection to b/156187905 not verified

### CODE REVIEW

File: tensorflow/python/eager/benchmarks_test.py [hunk 1]
```diff
@@ -476,6 +476,16 @@ def _benchmark_defun_matmul(self,
...
     self._run(func, num_iters, execution_mode=execution_mode)
 
+  def _benchmark_defun_matmul_with_signature(self, m, num_iters, execution_mode=None):
+    
+    def defun_matmul(m):
+      return math_ops.matmul(m, m)
+    f = function.defun(
+        defun_matmul, input_signature=[tensor_spec.TensorSpec([2, 2], dtypes.float32)])
+    
+    func = lambda: defun_matmul(m)
```
Comment (sun51) [inline tensorflow/python/eager/benchmarks_test.py:486]: this should be  f .

File: tensorflow/python/eager/benchmarks_test.py [hunk 2]
```diff
@@ -476,6 +476,16 @@ def _benchmark_defun_matmul(self,
     func = lambda: f(m, m, transpose_b=transpose_b)
     self._run(func, num_iters, execution_mode=execution_mode)
 
+  def _benchmark_defun_matmul_with_signature(self, m, num_iters, execution_mode=None):
+    
+    def defun_matmul(m):
```
Comment (sun51) [inline tensorflow/python/eager/benchmarks_test.py:481]: this is not really a defun, let us name it  func_matmul

File: tensorflow/python/eager/benchmarks_test.py [hunk 3]
```diff
@@ -577,6 +587,18 @@ def benchmark_defun_matmul_2_by_2_CPU(self):
       self._benchmark_defun_matmul(
           m, transpose_b=False, num_iters=self._num_iters_2_by_2)
 
+  @test_util.disable_tfrt("With signature")
```
Comment (sun51) [inline tensorflow/python/eager/benchmarks_test.py:590]: this is not needed. this benchmark should be able to run on TFRT.

File: tensorflow/python/eager/benchmarks_test.py [hunk 4]
```diff
@@ -652,6 +674,14 @@ def benchmark_defun_matmul_2_by_2_GPU(self):
...
```
Comment (sun51) [inline tensorflow/python/eager/benchmarks_test.py:677]: @test_util.disable_tfrt("copy to GPU not supported"). this string describes the reason why tfrt is disabled.

**Review by alextp — DISMISSED** [review]

**Comment by gbaned** [comment]
@jonathanchu33  Can you please address Ubuntu Sanity errors? Thanks!

**Review by sun51 — APPROVED** [review]

**Pull request merged**